app/ide-desktop/lib/dashboard/src/authentication/src/components/app.tsx

```diff
@@ -79,26 +80,16 @@ export const SET_USERNAME_PATH = '/set-username'
 // === App ===
 // ===========
 
-interface BaseAppProps {
+/** Global configuration for the `App` component. */
+export interface AppProps {
     logger: loggerProvider.Logger
     platform: platformModule.Platform
     /** Whether the dashboard should be rendered. */
     showDashboard: boolean
+    ide?: app.App
     onAuthenticated: () => void
 }
```

Comment:
This interface is used for properties for main react component by convention also named `App`. 

btw @somebody1234 do we use this `ide?` attribute at all?

Reply:
oh, sorry about this - before we switched to `runProject`/`runEnso`/`runApp` i was thinking of passing the `app.App` through to the dashboard, since the dashboard needed a way to create a new project. it should no longer be needed now.

Reply:
@PabloBuchu your reply does not answer my worry. If a struct contains "app" it should not be named "AppProps", no matter what. If it has to contain "app", it has to be renamed to reflect it's purpose.